uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -468,8 +468,10 @@ define([
       if (vm.isMode('view') || isLeaveType('toil')) { return; }
 
       _.each(['from', 'to'], function (type) {
-        $scope.$watch('detailsTab.uiOptions.times.' + type + '.amount', function (time, oldTime) {
-          return (time !== oldTime) && calculateBalanceChange();
+        $scope.$watch('detailsTab.uiOptions.times.' + type + '.amount', function (amount, oldAmount) {
+          if (isCalculationUnit('hours') && +amount !== +oldAmount) {
```

Comment:
Is is possible to have the unit in `minute`? then it should be taken into account as well? Would it be more appropriate to check `!isCalculationUnit('days')`?

Reply:
Well, this watcher is specific to hours, and we have only "hours" and "days". I can change though.